static/app/types/event.tsx

```diff
@@ -395,6 +395,7 @@ interface EventBase {
   device?: Record<string, any>;
   endTimestamp?: number;
   groupID?: string;
+  issueCategory?: 'performance' | 'error';
```

Comment:
Should we use the `IssueCategory` type here?

Reply:
Yup! tbh didn't know that was a type. Thanks!